app/controllers/organisations_controller.rb

```diff
@@ -85,6 +85,7 @@ def update
     if @organisation.update_attributes_with_superadmin(update_params)
       redirect_to @organisation, notice: 'Organisation was successfully updated.'
     else
+      @categories_start_with = Category.first_category_name_in_each_type
```

Comment:
is this a bug fix for something else? not sure how this is related to main feature